app/uk/gov/hmrc/customs/managesubscription/connectors/EmailConnector.scala

```diff
@@ -33,13 +33,13 @@ class EmailConnector @Inject() (appConfig: AppConfig, httpClient: HttpClient) {
   def sendEmail(email: Email)(implicit hc: HeaderCarrier): Future[HttpResponse] =
     httpClient.doPost[Email](appConfig.emailServiceUrl, email, Seq("Content-Type" -> "application/json")).map {
       response =>
-        logResponse(email.templateId)(response.status)
+        logResponse(email.templateId, response)
         response
     }
 
-  private def logResponse(templateId: String): Int => Unit = {
+  private def logResponse(templateId: String, response: HttpResponse) = response.status match {
     case ACCEPTED | OK => logger.info(s"sendEmail succeeded for template Id: $templateId")
-    case status        => logger.info(s"sendEmail: request is failed with status $status for template Id: $templateId")
+    case _             => logger.warn(s"sendEmail: request is failed with $response for template Id: $templateId")
```

Comment:
Hmm, I'm wondering what will be inside this response. Because warn logs will be visible on the production.

Reply:
Good point.  I thought we said that we wanted more info as to why the request failed?  Should I change it back to just logging status?

Reply:
I think that we can keep this as it is. Or we can change the `warn` to `info`.

But I think for now it's okay. The response shouldn't have any PII etc